### PR TITLE
Feature/remove static universe

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: examples/src/main/java/com/radixdlt/client/examples/ChatBot.java [hunk 1]
```diff
@@ -63,12 +63,10 @@ public Completable apply(DecryptedMessage message) {
 	}
 
 	public static void main(String[] args) throws Exception {
-		RadixUniverse.bootstrap(Bootstrap.BETANET);
```
Comment (flotothemoon) [inline examples/src/main/java/com/radixdlt/client/examples/ChatBot.java]: Yes! YES! Finally! :D

File: examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java [hunk 2]
```diff
@@ -33,7 +27,7 @@ public static void main(String[] args) throws Exception {
 			radixIdentity = RadixIdentities.loadOrCreateFile("my.key");
 		}
 
-		RadixApplicationAPI api = RadixApplicationAPI.create(radixIdentity);
+		RadixApplicationAPI api = RadixApplicationAPI.create(Bootstrap.LOCALHOST, radixIdentity);
```
Comment (flotothemoon) [inline examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java]: Some inconsistent formatting with the other example

File: radixdlt-java/src/main/java/com/radixdlt/client/atommodel/Ownable.java [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 
+import com.radixdlt.client.atommodel.accounts.RadixAddress;
 import com.radixdlt.client.core.crypto.ECPublicKey;
 
 /**
  * Temporary interface for representing a particle that can be owned and only invalidated by an owner.
  */
 public interface Ownable {
 	ECPublicKey getOwner();
+	RadixAddress getAddress();
```
Comment (flotothemoon) [inline radixdlt-java/src/main/java/com/radixdlt/client/atommodel/Ownable.java]: Hmm I think it's weird we have both, we shouldn't need both - can we remove getAddress() and just make `getOwner` return `RadixAddress`

File: radixdlt-java/src/main/java/com/radixdlt/client/core/Bootstrap.java [hunk 4]
```diff
@@ -2,19 +2,28 @@
...
 public enum Bootstrap implements BootstrapConfig {
-	BETANET(
+	LOCALHOST(
 		RadixUniverseConfigs::getBetanet,
 		Observable.just(
 			new RadixNode("localhost", false, 8080),
 			new RadixNode("localhost", false, 8081)
 		)
 	),
+	LOCALHOST_SINGLENODE(
```
Comment (flotothemoon) [inline radixdlt-java/src/main/java/com/radixdlt/client/core/Bootstrap.java]: Why are these different?

Comment (talekhinezh) [reply radixdlt-java/src/main/java/com/radixdlt/client/core/Bootstrap.java]: SINGLE_NODE will always connect to the same node so tests which have two clients can be synced easily

**Pull request merged**